task5,1/task5,1/Program.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+
+namespace homework_5._1
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Введите текст: ");
+            string lineRus = Console.ReadLine();
+
+            string lineEng = Transliteration(lineRus);
+            Console.WriteLine(lineEng);
+
+            Console.ReadKey();
+
+        }
+        static string Transliteration(string str)
+        {
+            str = str.Replace("А", "A");
```

Comment:
Лучше перед заменой перевести строку в верхний регистр. Тогда придется писать в два раза меньше кода.
```suggestion
            str = str.ToUpper();
            str = str.Replace("А", "A");
```